src/main/steps/home/get.ts

```diff
@@ -86,16 +86,11 @@ const applicant1RedirectPageSwitch = (caseState: State, userCase: Partial<Case>,
           : CONTINUE_WITH_YOUR_APPLICATION
         : HUB_PAGE;
     }
-    case State.AwaitingAos:
-    case State.AwaitingConditionalOrder:
-    case State.AosDrafted:
-    case State.AosOverdue:
-    case State.Holding:
-    case State.AwaitingGeneralConsideration: {
-      return HUB_PAGE;
+    case State.Draft: {
+      return isFirstQuestionComplete ? CHECK_ANSWERS_URL : YOUR_DETAILS_URL;
     }
     default: {
-      return isFirstQuestionComplete ? CHECK_ANSWERS_URL : YOUR_DETAILS_URL;
+      return HUB_PAGE;
```

Comment:
does this mean we can get rid of some of the get.tests, or is it worth still keeping them?

Reply:
we can keep it,
just noticed how big the other condition was getting so i felt like default should be hub page as really there are only a few states for this which is really draft as the others are convered.